server/games/game.py

```diff
@@ -120,8 +121,9 @@ def __init__(
         self._launch_fut = asyncio.Future()
 
         self._logger.debug("%s created", self)
+        asyncio.get_event_loop().create_task(self.timeout_game(setup_timeout))
 
-    async def timeout_game(self, timeout: int = 30):
+    async def timeout_game(self, timeout: int = 60):
```

Comment:
So this does change the timeout for custom games. I think that might cause 0 player games to hang around longer. You could fix this by adding the `setup_timeout` parameter to the `new_kwargs` dict in the custom game constructor

Reply:
Done